ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -539,21 +547,39 @@ trait AbstractHttpServiceIntegrationTestFuns extends StrictLogging {
       }
     }
 
-  protected def initialIouCreate(serviceUri: Uri): Future[(StatusCode, JsValue)] = {
-    val payload = TestUtil.readFile("it/iouCreateCommand.json")
+  protected def initialIouCreate(
+      serviceUri: Uri,
+      party: String = "Alice",
+      headers: List[HttpHeader] = headersWithAuth,
+  ): Future[(StatusCode, JsValue)] = {
+    val payload =
+      s"""
+        |{
+        |  "templateId": "Iou:Iou",
+        |  "payload": {
+        |    "observers": [],
+        |    "issuer": "$party",
+        |    "amount": "999.99",
+        |    "currency": "USD",
+        |    "owner": "$party"
+        |  }
+        |}
+        |""".stripMargin
     HttpServiceTestFixture.postJsonStringRequest(
       serviceUri.withPath(Uri.Path("/v1/create")),
       payload,
-      headersWithAuth,
+      headers,
     )
   }
 
   protected def initialAccountCreate(
       serviceUri: Uri,
       encoder: DomainJsonEncoder,
+      owner: String = "Alice",
```

Comment:
I would advice strongly against leaving `"Alice"` as a default. If we keep re-using the same ledger I'm fairly sure this would later cause some weird error and we can prevent some wasted time by getting rid of the default.

Reply:
Well I did go through the same pain , but it was partly due to the test passing using the jwt headers with `Alice` as default . I am happy to change this, can be done in a followup PR as I had left it out to reduce the footprint of the changes on this PR.